lib/rubydoop/dsl.rb

```diff
@@ -106,19 +106,31 @@ def input(paths, options={})
     #
     # Calls `setOutputFormatClass` on the Hadoop job and uses the static
     # `setOutputPath` on the output format to set the job's output path.
+    # Or if called without arguments, returns the output path of the job.
```

Comment:
Why not always return the output path? As it is this method doesn't return anything (or returns something unintentionally, not sure what `#set_output_format_class` returns).